File: Python/exercicio-08/solucao.py
def calcular_medidas():
    metros = float(input('Digite uma medida em metros: '))
    kilometros = metros / 1000
    hectometros = metros / 100
    decametros = metros / 10
    decimetros = metros * 10
    centimetros = metros * 100
    milimetros = metros * 1000
    print(f'A distancia de {metros}m corresponde a: \n {kilometros}Km \n {hectometros}Hm \n {decametros}Dam \n {decimetros}dm \n {centimetros}cm \n {milimetros}mm')
    print('Pressione Enter para sair')


calcular_medidas()

File: JavaScript/exercicio-08/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function converterMedidas() {
  const metros = parseFloat(prompt("Digite uma medida em metros " ));

  const kilometros = metros / 1000
  const hectometro = metros /  100
  const decametros = metros /   10
  const decimetros = metros *   10
  const centimetros = metros * 100
  const milimetros = metros * 1000

  console.log(`A distancia ${metros}m equivale a
  ${kilometros}km
  ${hectometro}hm
  ${decametros}dam
  ${decimetros}dm
  ${centimetros}cm
  ${milimetros}mm`);

  readline.question('Digite Enter para sair ... ');
}

converterMedidas();